ivy/functional/frontends/numpy/mathematical_functions/extrema_finding.py

```diff
@@ -51,3 +50,41 @@ def amin(
         a = ivy.concat([a, header], axis=axis)
 
     return ivy.min(a, axis=axis, keepdims=keepdims, out=out)
+
+
+def nanmin(
+    a,
+    axis=None,
+    out=None,
+    keepdims=False,
+    initial=None,
+    where=True,
+):
+    nan_mask = ivy.isnan(a)
+    a = ivy.where(ivy.logical_not(nan_mask), a, ivy.default(out, a.full_like(+ivy.inf)))
```

Comment:
This introduces the out values before taking the min, should be `a = ivy.where(ivy.logical_not(nan_mask), a, a.full_like(+ivy.inf))`